zwift_offline.py

```diff
@@ -466,7 +466,7 @@ def forgot():
                         message['From'] = sender_email
                         message['To'] = username
                         message['Subject'] = "Password reset"
-                        content = "https://launcher.zwift.com/login/?token=%s" % (user.get_token())
+                        content = "https://%s/login/?token=%s" % (server_ip, user.get_token())
```

Comment:
since we're most likely replacing it with an ip address, would it be better to use http instead?

Reply:
http would be handled by `CDNHandler(SimpleHTTPRequestHandler)` and we need it to be handled by Flask.

I know using the IP address instead of a hostname will cause a browser security warning, but the user may open the e-mail on a device which doesn't have the hostname redirected to the zoffline server.

Also notice that this is a very rarely used feature, it would only be used on an "online" server with a properly configured gmail account (as explained in "extra optional steps" of the readme).